File: fix-types.py
#!/usr/bin/env python3
import re

# Read the file
with open('src/components/AdminPanel.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace patterns
content = re.sub(r'courses\.map\(c =>', 'courses.map((c: any) =>', content)
content = re.sub(r'lessons\.map\(l =>', 'lessons.map((l: any) =>', content)

# Write back
with open('src/components/AdminPanel.tsx', 'w', encoding='utf-8') as f:
    f.write(content)

print("Fixed AdminPanel.tsx")

File: fix-types.js
const fs = require('fs');

// Read the file
let content = fs.readFileSync('src/components/AdminPanel.tsx', 'utf8');

// Replace patterns
content = content.replace(/courses\.map\(c =>/g, 'courses.map((c: any) =>');
content = content.replace(/lessons\.map\(l =>/g, 'lessons.map((l: any) =>');

// Write back
fs.writeFileSync('src/components/AdminPanel.tsx', content, 'utf8');

console.log('✅ Fixed AdminPanel.tsx');
